packages/plugin-html-vas-response/README.md

```diff
@@ -0,0 +1,27 @@
+# html-vas-response
+
+## Overview
+
+This plugin collects responses to an arbitrary HTML string using a point-and-click visual analogue scale.
+
+## Loading
+
+```js
+<script src="plugin-html-vas-response/index.js">
```

Comment:
This could reference the CDN link that will be generated.

Reply:
Updated!